print-bgp.c

```diff
@@ -80,13 +80,15 @@ struct bgp_open {
     nd_uint16_t bgpo_holdtime;
     nd_uint32_t bgpo_id;
     nd_uint8_t  bgpo_optlen;
+    nd_uint8_t bgpo_opttype;
+    nd_uint16_t bgpo_optlen_extended;
     /* options should follow */
 };
 #define BGP_OPEN_SIZE        29    /* unaligned */
 
 struct bgp_opt {
     nd_uint8_t bgpopt_type;
-    nd_uint8_t bgpopt_len;
+    nd_uint16_t bgpopt_len;
```

Comment:
Please add a comment saying that this can be one or two bytes, depending on whether or not RFC9072 is in use.

Reply:
Fixed, thanks.